src/main/Application.java

```diff
@@ -0,0 +1,39 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) {
+        // TODO: 프로그램 구현
+        System.out.printf("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n");
+        Scanner sc = new Scanner(System.in);
+        String[] car = sc.nextLine().split(",");
+        System.out.printf("시도할 회수는 몇회인가요?\n");
+        int num = sc.nextInt();
+        int[] position = new int[car.length];
+        int max = 0;
```

Comment:
README.md에 작성되어있는 요구사항에 맞게 Console.readLine() 을 이용하여 입력을 받아주세요

```suggestion
        String[] car = Console.readLine().split(",");
        System.out.printf("시도할 회수는 몇회인가요?\n");
        int num = Integer.parseInt(Console.readLine());
        int[] position = new int[car.length];
        int max = 0;
```